OpenChange/RTFHandler.m

```diff
@@ -529,109 +763,137 @@ - (RTFColorTable *) parseColorTable
 //
 // We receive the full string.
 //
+
+
 - (RTFFontTable *) parseFontTable
 {
```

Comment:
To follow the same standards that Objective C has, using `[self parse]` would be great. See the following link for details:
https://developer.apple.com/library/ios/documentation/Cocoa/Reference/Foundation/Classes/NSXMLParser_Class/index.html#//apple_ref/occ/instm/NSXMLParser/parse